tests/ut/language/parser/test_manual_scope_parsing.py

```
@@ -70,34 +89,76 @@ def k2(self, x: pl.Tensor[[64], pl.FP32]) -> pl.Tensor[[64], pl.FP32]:
             @pl.function(type=pl.FunctionType.Orchestration)
             def main(self, x: pl.Tensor[[64], pl.FP32]) -> pl.Tensor[[64], pl.FP32]:
                 with pl.manual_scope():
-                    a = self.k1(x)
-                    b = self.k2(x, deps=[a])
+                    a, a_tid = pl.submit(self.k1, x)
```

Ruff (0.15.12)

[warning] Unpacked variable `a` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)